charts/budibase/values.yaml

```diff
@@ -103,6 +103,7 @@ globals:
   google:
     clientId: "" 
     secret: ""
+  automationMaxIterations: "0"
```

Comment:
should we give this some sort of default in the `values.yaml`? My thinking is for self hosting Kubernetes users - not just BB cloud.

Reply:
Yep you're right, what do you think is a reasonable max? 500? More? I'm really not sure.

Reply:
I think 500 would be reasonable for now, since this can perform async operations I think it'd be dangerous to allow too large a limit.